.github/workflows/ci.yml

```diff
@@ -295,9 +294,6 @@ jobs:
             libprotobuf-dev \
             protobuf-compiler \
             "linux-headers-$(uname -r)"
-          sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-14 90
```

Comment:
Ported CI to use standard clang and llvm present in the ubuntu-24.04 runner image.